pyproject.toml

```diff
@@ -0,0 +1,16 @@
+[tool.poetry]
+name = "cfa-config-generator"
+version = "0.1.0"
+description = "Config generation tool for modeling pipelines."
+authors = ["amondal2 <ab59@cdc.gov>"]
+license = "ASL"
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "^3.10"
+pre-commit = "^4.0.1"
```

Comment:
This one can probably be moved to a [development dependency](https://python-poetry.org/docs/managing-dependencies/#adding-a-dependency-to-a-group)

Reply:
Done!